pkg/pod/list.go

```diff
@@ -70,3 +71,25 @@ func ListInAllNamespaces(apiClient *clients.Settings, options v1.ListOptions) ([
 
 	return podObjects, nil
 }
+
+// WaitForAllPodsInNamespaceRunning check that all pods in namespace that match options are in running state
+func WaitForAllPodsInNamespaceRunning(
+	apiClient *clients.Settings,
+	nsname string,
```

Comment:
Please validate is nsname is not empty.


	if nsname == "" {
		glog.V(100).Infof('nsname' parameter can not be empty")
		return nil, fmt.Errorf("failed to list pods, 'nsname' parameter is empty")
	}